app/src/main/java/woowacourse/payments/newcard/NewCardScreen.kt

```
@@ -80,3 +99,37 @@ fun NewCardScreen(
+    password: String,
+): Card? {
+    return try {
+        val cardNumber = CardNumber(cardNumber)
+        val expiredMonth = expiredDate.extractMonth() ?: return null
+        val expiredYear = expiredDate.extractYear() ?: return null
+        val expiredDate = ExpiredDate.of(expiredMonth, expiredYear) ?: return null
+        val ownerName = OwnerName(ownerName)
+        val password = Password(password)
+
+        Card(cardNumber, expiredDate, ownerName, password)
+    } catch (e: Throwable) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)